br/pkg/lightning/backend/local/local.go

```diff
@@ -1047,8 +1052,8 @@ func (local *local) prepareAndGenerateUnfinishedJob(
 			log.FromContext(ctx).Warn("split and scatter failed in retry", zap.Stringer("uuid", engineUUID),
 				log.ShortError(err), zap.Int("retry", i))
 		}
+		logger.End(zap.ErrorLevel, err)
```

Comment:
Please add some other key information for good search, such as "split & scatter ranges result : %v"

Reply:
the message "split and scatter ranges" will be printed, `logger.End` will use the message of `Begin`